src/CommonITILObject.php

```diff
@@ -8992,34 +8992,42 @@ public static function showKanban($ID)
             ],
             'supported_filters'           => [
                 'title' => [
+                    'label' => __('title'),
```

Comment:
We used a dedicated context for descriptions, we should do the same for titles.

```suggestion
                    'label' => _x('filters', 'title'),
```

Reply:
ok, I'll add it everywhere